<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddResidueDialog</class>
 <widget class="QDialog" name="AddResidueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="residueTypeLabel">
     <property name="text">
      <string>Residue Type:</string>
     </property>
     <property name="buddy">
      <cstring>residueType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="residueType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="chainIdLabel">
     <property name="text">
      <string>Chain Id:</string>
     </property>
     <property name="buddy">
      <cstring>chainId</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="chainId">
     <property name="maxLength">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="insertPositionGroup">
     <property name="title">
      <string>Insert Position:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="afterSelected">
        <property name="text">
         <string>After Selected</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="startOfModel">
        <property name="text">
         <string>Start of Model</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="beforeSelected">
        <property name="text">
         <string>Before Selected</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="endOfModel">
        <property name="text">
         <string>End of Model</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="putAtGroup">
     <property name="title">
      <string>Put At:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="screenCenter">
        <property name="text">
         <string>Screen Center</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="alphaHelix">
        <property name="text">
         <string>Alpha Helix</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="bestFit">
        <property name="text">
         <string>Best Fit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="betaSheet">
        <property name="text">
         <string>Beta Sheet</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>residueType</zorder>
  <zorder>chainId</zorder>
  <zorder>putAtGroup</zorder>
  <zorder>insertPositionGroup</zorder>
  <zorder>residueTypeLabel</zorder>
  <zorder>chainIdLabel</zorder>
  <zorder>chainIdLabel</zorder>
 </widget>
 <tabstops>
  <tabstop>residueType</tabstop>
  <tabstop>chainId</tabstop>
  <tabstop>afterSelected</tabstop>
  <tabstop>beforeSelected</tabstop>
  <tabstop>startOfModel</tabstop>
  <tabstop>endOfModel</tabstop>
  <tabstop>screenCenter</tabstop>
  <tabstop>bestFit</tabstop>
  <tabstop>alphaHelix</tabstop>
  <tabstop>betaSheet</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddResidueDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddResidueDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
